WalletWasabi.Fluent/Views/Wallets/Send/ChangeAvoidanceSuggestionControl.axaml

```diff
@@ -0,0 +1,52 @@
+<UserControl xmlns="https://github.com/avaloniaui"
+             xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
+             xmlns:d="http://schemas.microsoft.com/expression/blend/2008"
+             xmlns:mc="http://schemas.openxmlformats.org/markup-compatibility/2006"
+             xmlns:vm="using:WalletWasabi.Fluent.ViewModels.Wallets.Send"
+             mc:Ignorable="d" d:DesignWidth="300" d:DesignHeight="450"
+             x:DataType="vm:ChangeAvoidanceSuggestionViewModel"
+             ClipToBounds="False"
+             x:CompileBindings="True"
+             x:Class="WalletWasabi.Fluent.Views.Wallets.Send.ChangeAvoidanceSuggestionControl">
+  <Panel>
+    <Border Name="PART_DecorationBorderUnselected" >
+      <Border.Transitions>
+        <Transitions>
+          <DoubleTransition Property="Opacity"
+                            Easing="{StaticResource FluentEasing}"
+                            Duration="0:0:0.25" />
+        </Transitions>
+      </Border.Transitions>
+    </Border>
+    <Border Name="PART_DecorationBorderHover" >
+      <Border.Transitions>
+        <Transitions>
+          <DoubleTransition Property="Opacity"
+                            Easing="{StaticResource FluentEasing}"
+                            Duration="0:0:0.25" />
+        </Transitions>
+      </Border.Transitions>
+    </Border>
+    <Border Name="PART_MainContentBorder">
+      <StackPanel Spacing="20">
+        <Panel MinHeight="80" MinWidth="100">
+          <StackPanel Spacing="10" VerticalAlignment="Top">
+            <Panel>
+              <PathIcon Data="{StaticResource paper_cash_regular}" Height="25" Width="25"
+                        Foreground="{DynamicResource SystemAccentColor}" />
+            </Panel>
+
+            <TextBlock TextWrapping="Wrap" Text="Avoid Blockchain change by:" TextAlignment="Center" />
```

Comment:
Why not combine `Avoid Blockchain change by:` and `Sending` text together?

Reply:
done.